spinta/formats/ascii/helpers.py

```diff
@@ -1,11 +1,207 @@
-from tabulate import tabulate
+from typing import List, Optional, Tuple
 
 
-def draw(buffer, name, tnum, width):
-    if tnum > 1:
-        if name:
-            yield f"\n\nTable {name} #{tnum}:\n"
+def get_widths(
+    rows,
+    cols: List[str],
+    max_value_length: int = 100,
+    max_col_width: Optional[int] = None,
+    rows_to_check: int = 200
+) -> Tuple[List[dict], dict]:
+    widths = {}
+    max_widths = {}
+    max_split_widths = {}
+    read_rows = []
+
+    for i, row in enumerate(rows):
+        read_rows.append(row)
+        if i >= rows_to_check:
+            break
+        else:
+            for col in cols:
+                value = row.get(col)
+                if value:
+                    value = str(value)
+                    if len(value) > max_value_length:
+                        value = value[:max_value_length] + "..."
+
+                    if '\n' in value:
```

Comment:
Use `str.splitlines`, there are multiple line separators in different platforms.